corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/web_form_session_spec.js

```diff
@@ -280,15 +270,6 @@ hqDefine("cloudcare/js/form_entry/spec/web_form_session_spec", function () {
             );
 
             beforeEach(function () {
-                // Setup HTML
-                try {
-                    affix('input#submit');
-                    affix('#content');
-                } catch (e) {
-                    // temporarily catch this error while we work out issues running
-                    // mocha tests with grunt-mocha. this passes fine in browser
```

Comment:
Removal of this implies that it may break the test when running in a browser. Is that a problem?

Reply:
It passes in my browser locally. I read that comment as saying that the try/catch was added because of the grunt tests, so I was worried that removing it would break the tests GitHub runs, but those are passing fine, too.

My guess is this behavior is no longer tested. I blamed the affix calls back to the point where touchforms was migrated into this repo, 8 or 9 years ago. The web apps UI has been entirely overhauled since then, so it's plausible this isn't relevant anymore.